tests/schedules_test.py

```diff
@@ -62,7 +62,7 @@ def setUpClass(cls):
             )
 
             # Pick up a random slice and dashboard
-            slce = db.session.query(Slice).all()[0]
+            slce = db.session.query(Slice).filter_by(slice_name="Participants").all()[0]
```

Comment:
Also did the fix in a separate PR: https://github.com/apache/incubator-superset/pull/10489